code/processes/wdb.q

```diff
@@ -195,11 +196,11 @@ endofday:{[pt]
 	};
 	
 endofdaysave:{[dir;pt]
-	/- save remaining table rows to disk
-	.lg.o[`save;"saving the ",(", " sv string tl:tablelist[],())," table(s) to disk"];
-	savetables[dir;pt;1b;] each tl;
-	.lg.o[`savefinish;"finished saving data to disk"];
-	};
+    /- save remaining table rows to disk
+    .lg.o[`save;"saving the ",(", " sv string tl:tablelist[],())," table(s) to disk"];
+    savetables[dir;pt;1b;] each tl;
+    .lg.o[`savefinish;"finished saving data to disk"];
+  };
```

Comment:
these lines are changed from tabs to spaces but otherwise unchanged - I'd probably leave this out of this PR as it seems unrelated (and a lot of this file still appears to use tabs - don't think we have a consistent standard for tabs v spaces in TorQ)